java/gazelle/javaconfig/config.go

```diff
@@ -149,6 +155,7 @@ func New(repoRoot string) *Config {
 		extensionEnabled:       true,
 		isModuleRoot:           false,
 		generateProto:          true,
+		includeBinary:          true,
```

Comment:
How do you feel about renaming it to `generateBinaryTarget` / `generate_binary_target`?

As a user, "include" makes me think of file inclusion, and I feel the suggestion above is a bit more self-documenting. Also, we have a `generateProto` above, so it feels a tad more consistent. Just a suggestion :-)